src/js/media-controller.js

```diff
@@ -1283,7 +1283,18 @@ const volumeSupportPromise = hasVolumeSupportAsync().then((supported) => {
 
 const airplaySupported = !!window.WebKitPlaybackTargetAvailabilityEvent;
 const castSupported = !!window.chrome;
-const fullscreenEnabled = document[fullscreenApi.enabled];
+
+export const hasFullscreenSupport = (mediaEl = getTestMediaEl()) => {
+  let fullscreenEnabled = document[fullscreenApi.enabled];
+
+  if (!fullscreenEnabled) {
+    fullscreenEnabled = 'webkitDisplayingFullscreen' in mediaEl;
```

Comment:
is this the right way?
not https://developer.apple.com/documentation/webkitjs/htmlvideoelement/1628805-webkitsupportsfullscreen